lips/benchmark/airfransBenchmark.py

```diff
@@ -164,6 +166,15 @@ def evaluate_simulator(self,
 
         """
         self.augmented_simulator = augmented_simulator
+
+        field_names = self.train_dataset._attr_y
+        chunk_sizes = [int(simulation[1]) for simulation in self.train_dataset.extra_data["simulation_names"]]
+        flattened_train = np.concatenate([self.train_dataset.data[field_name][:, None] for field_name in field_names], axis = 1)
+        mean_observ,std_observ = iterative_fit(flattened_train,chunk_sizes)
+        self.ml_normalization["mean"] = mean_observ
+        self.ml_normalization["std"] = std_observ
+        self.evaluation.ml_normalization = self.ml_normalization
```

Comment:
Following my comments in `airfransEvaluation`, for the sake of generality, you could pass the ml_normalization as an argument to the evaluation instead of changing the class attribute. 

For sure, it is just my idea, I let you decide the way you prefer.